VMBackup/main/fsfreezer.py

```diff
@@ -205,20 +206,9 @@ def freeze_safe(self,timeout):
             end_time = datetime.datetime.utcnow()
             self.logger.log("Wait time to aquire lock "+ str(end_time - start_time),True)
 
-            
             sig_handle = None
             if (self.isAquireLockSucceeded == True):
                 sig_handle=self.freeze_handler.startproc(args)
-                self.thaw_safe()
```

Comment:
was this a bug ?

Reply:
@vityagi - yeah...though got pushed 2 days back only. I found it while I was testing my code optimization changes and asked arvind to do needful changes